buildmaster/old_new_porting_map.yml

```
@@ -0,0 +1,388 @@
+# Mapping between old and new commondata for oldcommondata_porter
+#
+# <old_commondata_name>:
+#   new_name: <new_name>
+#   reference_arxiv: <http://etc>
+#   reference_hepdata: <http://etc>
+#
+
+# HERA
+HERACOMBCCEM:
+  new_name: HERA_CC_318GEV_EM-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t7
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBCCEP:
+  new_name: HERA_CC_318GEV_EP-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t6
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEM:
+  new_name: HERA_NC_318GEV_EM-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t5
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEP460:
+  new_name: HERA_NC_225GEV_EP-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t4
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEP575:
+  new_name: HERA_NC_251GEV_EP-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t3
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEP820:
+  new_name: HERA_NC_300GEV_EP-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t2
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMBNCEP920:
+  new_name: HERA_NC_318GEV_EP-SIGMARED
+  reference_arxiv: '1506.06042'
+  reference_hepdata: 10.17182/hepdata.68951.v1/t1
+  journal: Eur. Phys. J. C75 (2015) 580
+HERACOMB_SIGMARED_B:
+  new_name: HERA_NC_318GEV_EAVG_BOTTOM-SIGMARED
+  reference_arxiv: '1804.01019'
+  reference_hepdata: ''
+  journal: Eur. Phys. J. C78 (2018) 473
+HERACOMB_SIGMARED_C:
+  new_name: HERA_NC_318GEV_EAVG_CHARM-SIGMARED
+  reference_arxiv: '1804.01019'
+  reference_hepdata: ''
+  journal: Eur. Phys. J. C78 (2018) 473
+
+## ATLAS
+ATLASLOMASSDY11EXT:
+  new_name: ATLAS_Z0_7TEV_LOMASS_M
+  reference_arxiv: '1404.1212'
+  reference_hepdata: 10.17182/hepdata.64183.v1/t6
+  journal: JHEP 06 (2014) 112
+
+ATLASWRAP11CC:
+  new_name: ATLAS_WPWM_7TEV_CC
```

```suggestion
  new_name: ATLAS_WPWM_7TEV_CC-ETA
```
Or even `new_name: ATLAS_WPWM_7TEV_CC_ETA` as you prefer.